memory_agents/youtube_trend_agent/.env.example

```
@@ -1,3 +1,13 @@
-OPENAI_API_KEY=Your Minimax api key here
+# Nebius Token Factory API key (used via the OpenAI-compatible SDK).
+NEBIUS_API_KEY=Your Nebius API key here
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)